reasoner_pydantic/kgraph.py

```diff
@@ -58,6 +65,7 @@ class Edge(BaseModel):
         title="object node id",
     )
     predicate: Optional[BiolinkPredicate] = Field(None, nullable=True)
+    qualifiers: Optional[HashableSet[Qualifier]] = Field(None, nullable=True)
```

Comment:
When we add new fields we must also modify update() accordingly. Technically we don't necessarily need logic for how qualifiers will be merged, because by all accounts you should only ever merge (update) edges that have the same qualifiers. We don't have the custom hashing for edges in this branch yet so it's not clear to me that we need the attribute logic that is currently present. 

I think this comment is primarily for awareness and I am not specifically asking for changes.

Reply:
We definitely do need more robust testing for qualifiers and constraints, and for the models in general. Like you said, we don't actually need any qualifiers logic, because edges should only be merged if they share the same exact qualifiers, so that didn't need to be updated.